application/Config/Autoload.php

```diff
@@ -50,12 +50,20 @@ public function __construct()
 		 *   `];
 		 */
 		$psr4 = [
-			'Config'                     => APPPATH.'Config',
-			APP_NAMESPACE                => APPPATH,			    // For custom namespace
-			'App'                        => APPPATH,			    // To ensure filters, etc still found,
-			'Tests\Support'              => TESTPATH.'_support',    // So custom migrations can run during testing
+			'Config'      => APPPATH . 'Config',
+			APP_NAMESPACE => APPPATH,                // For custom namespace
+			'App'         => APPPATH,                // To ensure filters, etc still found,
 		];
 
+		if (defined('ENVIRONMENT') && ENVIRONMENT === 'testing')
+		{
+			$psr4 = array_merge(
```

Comment:
Why `array_merge` here? Seems like just adding the element directly to the array is more straightforward: 

```
$psr4['Tests\Support'] = TESTPATH .'_support';
```

Reply:
implemented